invenio_vocabularies/datastreams/readers.py

```diff
@@ -224,19 +224,31 @@ def _iter(self, fp, *args, **kwargs):
 class XMLReader(BaseReader):
     """XML reader."""
 
+    def __init__(self, root_element=None, *args, **kwargs):
+        """Constructor."""
+        # TODO: How to make root_element mandatory?
+        self.root_element = root_element
```

Comment:
Can we just use it as an `arg` instead of the keyword, and remove the `=None`?

Reply:
In most of the readers this is how the reader specific props are being added 
e.g. [SimpleHTTPReader](https://github.com/inveniosoftware/invenio-vocabularies/blob/cf985f918a1fdfdee1486a2646ed82f6c78a80f1/invenio_vocabularies/datastreams/readers.py#L104), [ZipReader](https://github.com/inveniosoftware/invenio-vocabularies/blob/cf985f918a1fdfdee1486a2646ed82f6c78a80f1/invenio_vocabularies/datastreams/readers.py#L134), [CSVReader](https://github.com/inveniosoftware/invenio-vocabularies/blob/cf985f918a1fdfdee1486a2646ed82f6c78a80f1/invenio_vocabularies/datastreams/readers.py#L207)
Also addressed here: https://github.com/inveniosoftware/invenio-vocabularies/pull/402/files#r1778204039